src/server/JasmineGraphInstanceService.cpp

```diff
@@ -80,4301 +144,5243 @@ void *instanceservicesession(void *dummyPt) {
-                Utils::getJasmineGraphProperty("org.jasminegraph.server.instance.datafolder") + "/" + fileName;
-            int fileSize = atoi(size.c_str());
-            while (Utils::fileExists(fullFilePath) && Utils::getFileSize(fullFilePath) < fileSize) {
-                bzero(data, INSTANCE_DATA_LENGTH + 1);
-                read(connFd, data, INSTANCE_DATA_LENGTH);
-                line = (data);
+        bzero(data, 301);
+        int result_wr = write(sockfd, JasmineGraphInstanceProtocol::WORKER_EGO_NET.c_str(),
+                              JasmineGraphInstanceProtocol::WORKER_EGO_NET.size());
 
-                if (line.compare(JasmineGraphInstanceProtocol::FILE_RECV_CHK) == 0) {
-                    write(connFd, JasmineGraphInstanceProtocol::FILE_RECV_WAIT.c_str(),
-                          JasmineGraphInstanceProtocol::FILE_RECV_WAIT.size());
-                }
-            }
+        if (result_wr < 0) {
+            instance_logger.log("Error writing to socket", "error");
+        }
 
-            bzero(data, INSTANCE_DATA_LENGTH + 1);
-            read(connFd, data, INSTANCE_DATA_LENGTH);
-            line = (data);
+        instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::WORKER_EGO_NET, "info");
 
-            if (line.compare(JasmineGraphInstanceProtocol::FILE_RECV_CHK) == 0) {
-                instance_logger.log("Received : " + JasmineGraphInstanceProtocol::FILE_RECV_CHK, "info");
-                write(connFd, JasmineGraphInstanceProtocol::FILE_ACK.c_str(),
-                      JasmineGraphInstanceProtocol::FILE_ACK.size());
-                instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::FILE_ACK, "info");
-            }
-            if (totalPartitions != 0) {
-                JasmineGraphInstanceService::collectTrainedModels(sessionargs, graphID, graphPartitionedHosts,
-                                                                  totalPartitions);
-            }
-            std::vector<std::string> predictargs;
-            predictargs.push_back(graphID);
-            predictargs.push_back(vertexCount);
-            predictargs.push_back(fullFilePath);
-            predictargs.push_back(
-                Utils::getJasmineGraphProperty("org.jasminegraph.server.instance.trainedmodelfolder"));
-            predictargs.push_back(to_string(totalPartitions + stoi(ownPartitions)));
-            std::vector<char *> predict_agrs_vector;
-            std::transform(predictargs.begin(), predictargs.end(), std::back_inserter(predict_agrs_vector), converter);
-
-            std::string path = "cd " + Utils::getJasmineGraphProperty("org.jasminegraph.graphsage") + " && ";
-            std::string command = path + "python3.8 predict.py ";
-
-            int argc = predictargs.size();
-            for (int i = 0; i < argc; ++i) {
-                command += predictargs[i];
-                command += " ";
-            }
+        bzero(data, 301);
+        read(sockfd, data, 300);
```

Comment:
```suggestion
        read(sockfd, data, INSTANCE_DATA_LENGTH);
```

Reply:
All such occurrences are replaced in next PR

Reply:
Ok